CMakeLists.txt

```diff
@@ -17,9 +17,6 @@ enable_language(C CXX)
 
 set(BUILD_dev_dpmjetIII193 CACHE
   PATH "source directory for dev_dpmjetIII193")
-option(BUILD_sib23c00 OFF)
-option(BUILD_sib23c02 OFF)
-option(BUILD_sib23c03 OFF)
 
```

Comment:
sib23c03, ... will not be build at all. Or `if` conditions using BUILD_... variables  should be removed

Reply:
I went back and forth a lot with this PR and eventually forgot to remove the if (BUILD_sib23c00) etc. I now removed them. We don't need these ifs anymore, becaues the source code is available and the targets are just not build.